XenAdmin/MainWindow.resx

```diff
@@ -2311,130 +2311,139 @@
   </data>
   <data name="startOnHostToolStripMenuItem.Size" type="System.Drawing.Size, System.Drawing">
-    <value>242, 22</value>
+    <value>264, 22</value>
   </data>
   <data name="startOnHostToolStripMenuItem.Text" xml:space="preserve">
     <value>Start &amp;on Server</value>
   </data>
   <data name="toolStripSeparator20.Size" type="System.Drawing.Size, System.Drawing">
-    <value>239, 6</value>
+    <value>261, 6</value>
   </data>
   <data name="assignSnapshotScheduleToolStripMenuItem.Size" type="System.Drawing.Size, System.Drawing">
-    <value>242, 22</value>
+    <value>264, 22</value>
   </data>
   <data name="assignSnapshotScheduleToolStripMenuItem.Text" xml:space="preserve">
     <value>Assign to Snaps&amp;hot Schedule...</value>
   </data>
   <data name="assignToVirtualApplianceToolStripMenuItem.Size" type="System.Drawing.Size, System.Drawing">
-    <value>242, 22</value>
+    <value>264, 22</value>
   </data>
   <data name="assignToVirtualApplianceToolStripMenuItem.Text" xml:space="preserve">
     <value>Assign to &amp;vApp</value>
   </data>
   <data name="toolStripMenuItem9.Size" type="System.Drawing.Size, System.Drawing">
-    <value>239, 6</value>
+    <value>261, 6</value>
   </data>
   <data name="copyVMtoSharedStorageMenuItem.Size" type="System.Drawing.Size, System.Drawing">
-    <value>242, 22</value>
+    <value>264, 22</value>
   </data>
   <data name="copyVMtoSharedStorageMenuItem.Text" xml:space="preserve">
     <value>&amp;Copy VM</value>
   </data>
   <data name="MoveVMToolStripMenuItem.Size" type="System.Drawing.Size, System.Drawing">
-    <value>242, 22</value>
+    <value>264, 22</value>
   </data>
   <data name="MoveVMToolStripMenuItem.Text" xml:space="preserve">
     <value>&amp;Move VM</value>
   </data>
   <data name="snapshotToolStripMenuItem.Size" type="System.Drawing.Size, System.Drawing">
-    <value>242, 22</value>
+    <value>264, 22</value>
   </data>
   <data name="snapshotToolStripMenuItem.Text" xml:space="preserve">
     <value>Ta&amp;ke a Snapshot</value>
   </data>
   <data name="convertToTemplateToolStripMenuItem.Size" type="System.Drawing.Size, System.Drawing">
-    <value>242, 22</value>
+    <value>264, 22</value>
   </data>
   <data name="convertToTemplateToolStripMenuItem.Text" xml:space="preserve">
     <value>Convert to &amp;Template</value>
   </data>
   <data name="exportToolStripMenuItem.Size" type="System.Drawing.Size, System.Drawing">
-    <value>242, 22</value>
+    <value>264, 22</value>
   </data>
   <data name="exportToolStripMenuItem.Text" xml:space="preserve">
     <value>&amp;Export...</value>
   </data>
+  <data name="toolStripSeparator32.Size" type="System.Drawing.Size, System.Drawing">
+    <value>261, 6</value>
+  </data>
+  <data name="toolStripMenuItemVtpm.Size" type="System.Drawing.Size, System.Drawing">
+    <value>264, 22</value>
+  </data>
+  <data name="toolStripMenuItemVtpm.Text" xml:space="preserve">
+    <value>Trusted Plat&amp;form Module Manager...</value>

```

Comment:
The wording for various management dialogs is a bit mixed in XC, e.g. we use "Manage vApps..." and "Conversion Manager...". I went with "TPM Manager..." because Win10 uses "TPM Management" as the title of the relevant dialog, although "Manage vTPMs..." may be more descriptive of what our dialog does. Ok, I'll revisit this, will probably check with the server team too.